jenkins/param.yml

```diff
@@ -170,6 +170,7 @@
           - ubuntu:artful
```

Comment:
we should remove artful as we don't support it

Reply:
done